.chloggen/ottl-replace-capture-groups.yaml

```
@@ -0,0 +1,17 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
```

Technically this is a breaking change bc anyone using `$` today needs to update their config.

```suggestion
change_type: breaking
```